extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js

```diff
@@ -101,24 +102,23 @@ require(
 		}
 
 		function displayNotification(message) {
-			w.GlobalNotification.show(message, 'notify');
-
-			// Track a view of the notification
-			var trackingParams = {
-				trackingMethod: 'ga',
-				category: 'wikia-in-your-lang',
-				action: tracker.ACTIONS.VIEW,
-				label: targetLanguage + '-notification-view',
-			};
+			var notification = new BannerNotification(message, 'notify').show(),
+				// Track a view of the notification
+				trackingParams = {
+					trackingMethod: 'ga',
+					category: 'wikia-in-your-lang',
+					action: tracker.ACTIONS.VIEW,
+					label: targetLanguage + '-notification-view'
+				};
 
 			tracker.track(trackingParams);
 
 			// Bind tracking of clicks on a link and events on close
-			bindEvents();
+			bindEvents(notification.$element);
 		}
 
-		function bindEvents() {
-			$('.global-notification.notify')
+		function bindEvents($element) {
+			$element
 				.on('click', '.close', function() {
```

Comment:
This might be better handled as an `onClose` event that's emitted from the BannerNotification instance. That way we would be free to change stuff like the `.close` element without breaking stuff like this. 

Alternately, we could expose the `.close` element as `notification.$close` and bind to that.